### DISCUSSION TITLE
***question to users*** Python 3 / Ubuntu 20.04 success?

### DISCUSSION THREAD

**Discussion by jpreiss** [USC-ACTLab/crazyswarm#325] [General]
Hi all, we added Python 3 support a while back but I have not been able to test it personally. I am just wondering if anyone has successfully flown with Python 3 on Ubuntu 20.04. Thanks!

**Comment by whoenig** [comment]
I tested this and it does work very well.

**Reply by jpreiss** [reply]
Great!

**Reply by jpreiss** [reply]
@whoenig did you need to do any networking setup on Ubuntu 20 to get communication with your mocap machine working? For some reason my Ubuntu 20 install is no longer talking to Vicon even though I got it  to work in the past, and ubuntu 16 is still working.

**Reply by whoenig** [reply]
No, it just worked. The only difference I can think of is that this was Tracker 1.x with a pretty old VICON system (similar to what we had in the Cage). Otherwise it's a similar setup (two network cards - 1 for the cameras one for connection/Internet; connection to the Ubuntu PC using a switch).

**Reply by jpreiss** [reply]
Weirdly, upgrading to tracker 3.9 fixed it. I also had to address the machine as `vicon.local` instead of `vicon` in the launch file, but i also had to do that to even get `ping` working.
